### DISCUSSION TITLE
使用主题的时候网页异常的卡

### DISCUSSION THREAD

**Discussion by ehanblog** [MaLuns/hexo-theme-async#110] [Q&A 疑难杂症]
使用最新的主题在本地运行的时候，网页非常的卡，就像访问了一个国外的网站一样（但是开启了科学上网就不卡了）

**Comment by ehanblog** [comment]
![image](https://github.com/MaLuns/hexo-theme-async/assets/100177229/c182af48-9e97-48f3-99db-d7b17fbc015b)

**Reply by ehanblog** [reply]
这两个文件为什么是从网络上获取

**Comment by MaLuns** [comment]
三方插件 js 是从 CDN 加载的，可以通过配置切换 CDN 源，参考这里 https://hexo-theme-async.imalun.com/guide/config.html#%E6%8F%92%E4%BB%B6-plugins
